odk2-src/survey-form-processing.rst

```diff
@@ -581,13 +581,13 @@ The controller's progress through this program is tracked by the history stack m
         screenManager.setScreen(ctxt, screenOp, options);
     }
 
-i.e., the processing flow eventually calls `screenManager` to display a screen (via `setScreen(ctxt, screenOp, options)`) and perhaps also shows a pop-up with some sort of alert or error message (via `showScreenPopup(m)`).
+Processing flow, for example, eventually calls `screenManager` to display a screen (via `setScreen(ctxt, screenOp, options)`) and perhaps also shows a pop-up with some sort of alert or error message (via `showScreenPopup(m)`).
```

Comment:
There shouldn't be a 'for example' here, rewording it as 'Simply put, the processing flow eventually calls ...' will preserve the intended meaning.

Reply:
Great. I appreciate your feedback. I can switch it to "Simply put..." so that it preserves the meaning intended. Do I edit it and do another pull request on this branch? What is the protocol?

Reply:
Thanks. Hopefully this worked.